### PR TITLE
[Refactor]Remove unnecessary judgment

### PR DESCRIPTION
**Pull request by liugddx**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

Remove unnecessary judgment

## Brief changelog

update GrpcClient

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: common/src/main/java/com/alibaba/nacos/common/remote/client/grpc/GrpcClient.java [hunk 1]
```diff
@@ -71,18 +71,18 @@
  */
 @SuppressWarnings("PMD.AbstractClassShouldStartWithAbstractNamingRule")
 public abstract class GrpcClient extends RpcClient {
-    
+
```
Comment (KomachiSion) [inline common/src/main/java/com/alibaba/nacos/common/remote/client/grpc/GrpcClient.java:74]: Please use nacos code style to reformat code.

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: common/src/main/java/com/alibaba/nacos/common/remote/client/grpc/GrpcClient.java (hunk 1 above)
Comment (liugddx) [reply common/src/main/java/com/alibaba/nacos/common/remote/client/grpc/GrpcClient.java:74]: > Please use nacos code style to reformat code.

I try to format the code with nacos check style, but there is no problem, I think it should be that nacos check style does not restrict this

Comment (liugddx) [reply common/src/main/java/com/alibaba/nacos/common/remote/client/grpc/GrpcClient.java:74]: image
image

Comment (liugddx) [reply common/src/main/java/com/alibaba/nacos/common/remote/client/grpc/GrpcClient.java:74]: Maybe #11021 is a better way.

Comment (liugddx) [reply common/src/main/java/com/alibaba/nacos/common/remote/client/grpc/GrpcClient.java]: > Please use nacos code style to reformat code.

Done. PTAL

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**